app/scripts/modules/core/src/presentation/forms/inputs/TextInput.tsx

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 
 import { orEmptyString, validationClassName } from './utils';
 import { IFormInputProps, OmitControlledInputPropsFrom } from './interface';
+import get = Reflect.get;
```

Comment:
Good question. That was an auto-import by WebStorm. What's the right one?

Reply:
we typically use `import { get } from 'lodash'` — that said, if you move to having `prefix` as an actual field on `ITextInputProps` then you'll probably end up referencing it directly instead of using `get`.